src/lib/util/settings.d.ts

```diff
@@ -0,0 +1,66 @@
+export type SettingsObject = {
+	token?: string;
+	applicationId?: string;
+	clientSecret?: string;
+	colors?: ColorsSettingsObject;
+	defaultLocaleCode?: string;
+	managers?: string[];
+	database?: DatabaseSettingsObject;
+	lavalink?: LavalinkSettingsObject;
+	features?: FeaturesSettingsObject;
+};
+
+export type ColorsSettingsObject = {
+    success?: string;
+    neutral?: string;
+    warning?: string;
+    error?: string;
```

Comment:
How about `ColorResolvable` and not `string` then remove where you used `ColorResolvable` as type assertions.